packages/core/src/repositories/documentVersionsRepository/index.test.ts

```diff
@@ -47,7 +47,10 @@ describe('DocumentVersionsRepository', () => {
 
       const documents = result.unwrap()
       expect(documents).toHaveLength(2)
-      expect(documents.map((d) => d.path)).toEqual(['foo', 'bar'])
+      expect(documents.map((d) => d.path).sort()).toEqual(['bar', 'foo'])
+      expect(new Set(documents.map((d) => d.path))).toEqual(
+        new Set(['foo', 'bar']),
+      )
```

Comment:
git conflict! I fixed it already

Reply:
let's see who merges faster :trollface:

Reply:
I said first that I fixed it if that counts ;) I feel I always lost this game for some reason